app/models/organisation.rb

```diff
@@ -16,7 +16,7 @@ def validate_in_register?
 
   def self.fetch_organisations_from_register
     UseCases::Organisation::FetchOrganisationRegister.new(
-      organisations_gateway: Gateways::OrganisationRegisterGateway.new
+      govuk_organisations_register_gateway: Gateways::GovukOrganisationsRegisterGateway.new
```

Comment:
I'd leave the param "govuk_organisations_register_gateway" as just "organisations_gateway".
This leaves it open for us to inject a different gateway in the future